src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml

```diff
@@ -152,5 +153,13 @@
         <service id="translation.extractor" class="%translation.extractor.class%"/>
 
         <service id="translation.writer" class="%translation.writer.class%"/>
+
+        <!-- Cache -->
+        <service id="translation.cache.default" class="Symfony\Component\Translation\MessageCache" public="false">
+            <argument key="cache_dir">%kernel.cache_dir%/translations</argument> <!-- cache_dir -->
+            <argument>%kernel.debug%</argument> <!-- debug -->
+        </service>
+
+        <service id="translation.cache" alias="translation.cache.default" />
```

Comment:
Do we need this? We already set the alias in the extension class. And in case the alias is not set (e.g. because the option is not used), we can set `on-invalid="null"` on the `translation.cache` argument in line 48.